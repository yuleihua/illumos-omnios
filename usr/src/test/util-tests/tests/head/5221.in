5221 head shouldn't reuse stdin
