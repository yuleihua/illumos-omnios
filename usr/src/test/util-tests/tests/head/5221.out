==> 5221.in <==
5221 head shouldn't reuse stdin

==> /dev/stdin <==
Old
Bill Joy
Bug

Lasts Forever
